backend/database/versions/f6b189a1d9c1_create_cellxgene_tables.py

```diff
@@ -0,0 +1,56 @@
+"""create_cellxgene_tables
+
+Revision ID: f6b189a1d9c1
+Revises: 899656d37baa
+Create Date: 2020-07-30 12:03:18.275767
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+from sqlalchemy import Column, String, TIMESTAMP, func
+
+revision = 'f6b189a1d9c1'
+down_revision = '899656d37baa'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
```

Comment:
wouldn't a cellxgene user be the same thing as a corpora user according to the DB?